pkg/middleware/loggermw/logger.go

```diff
@@ -154,8 +154,36 @@ func errorLogParams(err error) []any {
 	}
 }
 
-var sensitiveQueryStrings = [...]string{
-	"auth_token",
+type queryCheck struct {
+	key     string
+	checker func(v url.Values) bool
+}
+
+var sensitiveQueryChecks = [...]queryCheck{
```

Comment:
This feels very fragile and asking for a can of worms I'm afraid :)

Reply:
Too advanced perhaps? 😅 Should probably discuss with backend group. 

If someone uses loki they could basically filter out unwanted details already. Is that the solution for anyone?